src/js/components/organisms/brands2/brands2.ts

```diff
@@ -0,0 +1,34 @@
+import { Vue, Component, Prop } from 'vue-property-decorator';
+import { BrandsProps } from './brands2.models';
+import WithRender from './brands2.vue.html';
+
+@WithRender
+@Component({
+    name: 'Brands2',
```

Comment:
Perhaps rename this to brands, and the one in legacy change that one to legacy brands?

Reply:
i renamed the component to `brands`, but kept the css class `.brands2` to avoid conflict with the styling of the html in the `src/SiteBundle/Resources/views/Homepage/index.html.twig` file. i'd rather not touch legacy